lib/counter.js

```diff
@@ -35,13 +36,21 @@ function Counter(name, help, labels) {
 	if(!validateLabelNames(labels)) {
 		throw new Error('Invalid label name');
 	}
+
+	if(register === undefined) {
+		register = true;
```

Comment:
I don't like that we mutate the register property here, create a new variable inside the scope instead